include/ltest-macros.lfe

```diff
@@ -87,6 +87,35 @@ A simple wrapper macro for defining the tear-down function in a fixture."
 ;;; Test case macros
 ;;;===================================================================
 
+(eval-when-compile
+  (defun is-named-tuple (t)
+    (progn ;(lfe_io:format "~p" (list t)) ;for debugging
+           (is-named-tuple1 t)))
+
+  ;Return true if we have (tuple "name"...) or #("Name"...)
+  (defun is-named-tuple1
+    ((t) (when (is_tuple t))
+      (if (io_lib:printable_list (element 1 t))
+        'true
+        'false))
+    ((t) (when (is_list t))
+      (if (andalso (== 'tuple (hd t))
```

Comment:
do you mean the printable_list(...) test? It is needed to differentiante from tuples like
{spawn, Tests} or {timeout, Tests} etc.